xsamtools/cram.py

```diff
@@ -42,6 +48,15 @@ def __str__(self):
             msg = f"Command '{self.cmd}' returned non-zero exit status {self.returncode}."
         return f"{msg}\n\nERROR: {self.stderr}"
 
+def get_crai_indices(crai):
+    crai_indices = []
+    with open(crai, "rb") as fh:
+        with gzip.GzipFile(fileobj=fh) as gzip_reader:
+            with codecs.getreader("ascii")(gzip_reader) as reader:
```

Comment:
Please use `io.TextIOWrapper` instead of codecs. It is [much faster](https://www.python.org/dev/peps/pep-0400/#textiowrapper-features).

Reply:
Sounds good!